README.md

```diff
@@ -158,6 +158,7 @@ $index->search('hobbit', ['limit' => 2, 'filters' => 'title = "The Hitchhiker\'s
 ## 🤖 Compatibility with MeiliSearch
 
 This package is compatible with the following MeiliSearch versions:
+- `v0.15.X`
```

Comment:
I don't know what to think about this, because the SDK is fully compatible, but not the tests... If you run your new test with v0.14.1 or older, they will fail... WDYT?

Reply:
I had the same thought. We will discuss it in meeting before I change (or not) all the PRs